src/app/home/form/form.component.html

```diff
@@ -0,0 +1,56 @@
+<div>
+  <div class="{{ formClass }}">
+    <div class="card"></div>
+    <div class="card">
+      <h1 class="title">Login</h1>
+      <form>
+        <div class="input-container">
+          <input type="text" id="#{label}" required="required" />
+          <label for="#{label}">Username</label>
+          <div class="bar"></div>
+        </div>
+        <div class="input-container">
+          <input type="password" id="#{label}" required="required" />
+          <mat-icon class="eyeIcon" (click)="(toggleVisiblity)"
+            >remove_red_eye</mat-icon
+          >
```

Comment:
Is this `(click)` property expected ? I can't see any function named `toggleVisiblity` in our component file?